src/commissaire/util/config.py

```diff
@@ -51,9 +57,92 @@ def _normalize_member_names(json_object):
     return normalized
 
 
+def etcd_client_args():
+    """
+    Assembles a keyword argument dictionary from environment variables,
+    suitable for passing to ``etcd.Client``.
+
+    The environment variables used are:
+
+    ``ETCD_MACHINES`` - comma-separated list of etcd service URLs
+    ``ETCD_TLSPEM`` - path of TLS client certificate - public key
+    ``ETCD_TLSKEY`` - path of TLS client certificate - private key
+    ``ETCD_CACERT`` - path of TLS certificate authority public key
+    ``ETCD_USERNAME`` - username used for basic auth
+    ``ETCD_PASSWORD`` - password used for basic auth
+    """
+    args = {
+        'cert': None,  # see below
+        'ca_cert': os.environ.get('ETCD_CACERT'),
+        'username': os.environ.get('ETCD_USERNAME'),
+        'password': os.environ.get('ETCD_PASSWORD')
+    }
+
+    try:
+        args['cert'] = (
```

Comment:
nit: May want to throw some ``logging.debug`` statements in. I can forsee someone starting a server up and things not working and having those statements in silent ``pass``es may be helpful.